src/material-experimental/mdc-theming/_all-theme.scss

```diff
@@ -20,6 +19,5 @@
   @include mat-progress-bar-theme-mdc($theme);
   @include mat-radio-theme-mdc($theme);
   @include mat-slide-toggle-theme-mdc($theme);
-  @include mat-slider-theme-mdc($theme);
```

Comment:
can you leave this commented out with a TODO so we don't forget to add it back

Reply:
Added a comment with a TODO, thanks